src/app/archive/archive.ml

```diff
@@ -9,6 +9,12 @@ let command_run =
     (let%map_open log_json = Flag.Log.json
      and log_level = Flag.Log.level
      and server_port = Flag.Port.Archive.server
+     and metrics_server_port =
+       flag "--metrics-port" ~aliases:["metrics-port"]
```

Comment:
should the alias be `-metrics-port` (I think the single dash is implicit, but mentioning it would be consistent with the other aliases)